src/com/flickity.js

```diff
@@ -92,7 +92,7 @@
 			slider = item;
 			scope = s;
 
-			if (slider.find(o.cellSelector).length === 1) {
+			if (slider.find(o.cellSelector).length > 1) {
```

Comment:
So if one, we do nothing ?

Reply:
Yep! You can see at the end of the `if` we add an "aborted class" and if none are found a simple log into the console.